### PR TITLE
feat: editor page title rework

### PR DESCRIPTION
**Pull request by naomi-lgbt**
Checklist:

- [X] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [X] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [X] My pull request targets the `main` branch of freeCodeCamp.
- [X] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

This removes the `learn` text from the title of the page when viewing a challenge.

While this made sense for the current naming convention, it breaks with the new `Learn X by Building Y` convention.

### CODE REVIEW

**Review by raisedadead — DISMISSED** [review]

**Comment by ojeytonwilliams** [comment]
@nhcarrigan do we want to delay this until we launch the practice projects?

**Comment by nhcarrigan** [comment]
IMHO this could already be creating funky sentence structure in i18n versions of the site where "Learn CSS Grid" might take a different order.

**Comment by raisedadead** [comment]
Whats up with tests?

**Comment by nhcarrigan** [comment]
Whoops. Got those updated.

File: client/src/templates/Challenges/classic/show.tsx
Comment (ojeytonwilliams) [inline client/src/templates/Challenges/classic/show.tsx:402]:
```suggestion
          <Helmet title={`${this.getBlockNameTitle()} | freeCodeCamp.org`} />
```
Out of scope for this PR, but shouldn't these be translated?

Comment (nhcarrigan) [reply client/src/templates/Challenges/classic/show.tsx:402]: freeCodeCamp.org isn't translated.

image

The block names are translated elsewhere (I assume via the frontmatter).

Comment (ojeytonwilliams) [reply client/src/templates/Challenges/classic/show.tsx:402]: It should be "HTML básico y HTML5", no?

**Review by ojeytonwilliams — DISMISSED** [review]
Putting aside the out-of-scope missing translation, this LGTM

File: client/src/templates/Challenges/classic/show.tsx
Comment (ojeytonwilliams) [reply client/src/templates/Challenges/classic/show.tsx:402]: Feel free to mark this resolved whenever you want to merge, Nick.  I just wanted to mention the issue before I forgot about it.

Comment (nhcarrigan) [reply client/src/templates/Challenges/classic/show.tsx:402]: Oh snap yes you're correct.

Comment (nhcarrigan) [reply client/src/templates/Challenges/classic/show.tsx]: I went ahead and fixed the translation issue in this PR too, since it's touching the same code.

**Review by ojeytonwilliams — COMMENTED** [review]
Do we want to remove Learn from all the titles?  It still appears in some places: https://www.freecodecamp.org/learn/information-security/information-security-projects/stock-price-checker 'Information Security Projects - Stock Price Checker | Learn | freeCodeCamp.org'

**Review by moT01 — COMMENTED** [review]
I would be fine adding this as-is. It does solve the problem of having two `Learn`'s right in a row, and the i18n issue of sentences potentially not making sense. As Oliver pointed out, it looks like three of the layouts still have `Learn` in them. I would be fine leaving those in or removing them. They are only part of the layouts that will not use the project based naming convention for now. So we could maybe remove them when that changes? (if we remember)

**Comment by ojeytonwilliams** [comment]
> (if we remember)

That's my concern.  If we remove them now we don't rely on our future selves and we can remove that key-value pair from the jsons.

**Review by moT01 — APPROVED** [review]
Well, I'm fine with it either way. I was initially leaning towards removing them here. Now, I'm on the fence. If we remove them now, it won't say 'Learn' in the title anywhere for those challenges. If we accidentally leave it in after renaming the projects, it won't look bad. I'll leave it up to you and Nick on what to do with it. I'm going to approve this in case we decide to just go with it.

**Review by ojeytonwilliams — APPROVED** [review]
LGTM :+1: 

The remaining 'learn' isn't a huge deal, so let's get this in.

**Pull request merged**
